deepchem/utils/test/test_dftutils.py

```diff
@@ -22,16 +20,16 @@ def test_dftutils():
             'basis': '6-311++G(3df,3pd)'
         }
     }
-    atomzs, atomposs = dqc.parse_moldesc(system["kwargs"]["moldesc"])
-    mol = Mol(**system["kwargs"])
+    atomzs, atomposs = parse_moldesc(system["kwargs"]["moldesc"])
+    mol = Mol(**system["kwargs"], spin=0.0)
```

Comment:
For some reason earlier spin was not required but in this case spin should be provided.
- As per assert statements, fractional charge molecules should have spin parameter defined,